<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDatabaseDialog</class>
 <widget class="QWidget" name="ImportDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="vboxContent">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_file">
     <layout class="QHBoxLayout" name="layout_file">
      <item>
       <widget class="QLabel" name="lblFile">
        <property name="text">
         <string>导入数据库表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editTableFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectFile">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_mode">
     <layout class="QHBoxLayout" name="layout_mode">
      <item>
       <widget class="QLabel" name="lblMode">
        <property name="text">
         <string>数据库导入方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbIncrement">
        <property name="text">
         <string>增量</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_dup">
     <layout class="QHBoxLayout" name="layout_dup">
      <item>
       <widget class="QLabel" name="lblDup">
        <property name="text">
         <string>重复名处理方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSkip">
        <property name="text">
         <string>跳过</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOverwrite">
        <property name="text">
         <string>覆盖</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkUseDefaultHint">
     <property name="text">
      <string>使用默认值的提示</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
